src/main/haskell/kore/test/Test/Kore/ASTUtils/AlphaCompare.hs

```diff
@@ -0,0 +1,60 @@
+{-|
```

Comment:
I suppose that this is random-ish, but, when I checked test coverage, ctx2 was not evaluated in almost all functions, while next, rewrites, charliteral and stringliteral were not covered at all. Could you add tests that cover the entire code?

Reply:
That's just quickcheck not generating enough random terms.

`ctx1` and `ctx2` are only evaluated when the function has to compare a variable, and `ctx2` is not evaluated if the first variable being compared is free.